src/Symfony/Bundle/SecurityBundle/Resources/config/collectors.xml

```diff
@@ -10,7 +10,7 @@
             <argument type="service" id="security.token_storage" on-invalid="ignore" />
             <argument type="service" id="security.role_hierarchy" />
             <argument type="service" id="security.logout_url_generator" />
-            <argument type="service" id="debug.security.access.decision_manager" />
+            <argument type="service" id="security.access.decision_manager" />
```

Comment:
@yceruto the service is decorated the debug is on by loading `security_debug.xml`.